examples/nest/Potjans_2014/spike_analysis.py

```diff
@@ -57,16 +57,16 @@
 for l in gidfile:
     a = l.split()
     gids.append([int(a[0]), int(a[1])])
-print 'Global IDs:'
-print gids
-print
+print('Global IDs:')
+print(gids)
+print()
 
 # number of populations
 
 num_pops = len(gids)
-print 'Number of populations:'
-print num_pops
-print
+print('Number of populations:')
```

Comment:
I agree with you in principle, but in this case I prioritised getting things done over doing them perfectly. Example beautification can come in a separate PR later.